### PR TITLE
Address CL2.0 comments

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/CivicSubscriptionView/CivicSubscriptionView.vue [hunk 1]
```diff
@@ -325,6 +327,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    initialState: {
+      type: String,
+      default: undefined,
```
Comment (williamchong) [inline src/components/CivicSubscriptionView/CivicSubscriptionView.vue:332]: Strange decision to not set a default value for state, setting one should always be desirable?
Or maybe we need a state representing "undefined"

File: src/pages/_id/index.vue [hunk 2]
```diff
@@ -296,7 +296,9 @@ export default {
           creator,
           civicSupport,
           isShowCivicWelcome:
-            store.getters.getUserId && query.civic_welcome === '1',
+            store.getters.getUserId &&
+            civicSupport.quantity > 0 &&
```
Comment (williamchong) [inline src/pages/_id/index.vue:300]: `civicSupport && civicSupport.quantity > 0` seems more robust

File: src/pages/civic/payment/stripe/success.vue [hunk 3]
```diff
@@ -36,10 +39,13 @@ export default {
         'CivicRegisterComplete(stripe)',
         1
       );
+      const { from } = this.$route.query;
+      if (from && !this.getIsFollowedAuthor(from)) {
+        this.followAuthor(from);
```
Comment (williamchong) [inline src/pages/civic/payment/stripe/success.vue:44]: `followAuthor` was not defined in this page's `mapAction`

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/pages/civic/_from.vue [hunk 4]
```diff
@@ -1,311 +1,10 @@
...
-              v-model="selectedPaymentMethod"
-            )
-              option(
-                v-for="method in paymentMethods"
-                :key="method"
-                :value="method"
-              )
-                | {{ $t(`CivicPage.paymentMethod.${method}`) }}
-</template>
-
```
Comment (williamchong) [inline src/pages/civic/_from.vue:54]: Sometimes a page without template will cause server error, maybe leave some placeholder template?

File: src/components/CreatorsPage/sections/Calculator/Calculator.vue [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  },
+  computed: {
+    subscriptionEarnings() {
+      return this.subscribersCount * this.subscriptionPrice * 0.95;
+    },
+    formattedSubscriptionEarnings() {
+      return `$${this.subscriptionEarnings}`;
+    },
+    creatorsFundEarningsMin() {
+      return this.readersCount * 0.009 + this.readersCount * 0.01;
```
Comment (williamchong) [inline src/components/CreatorsPage/sections/Calculator/Calculator.vue:107]: what is the meaning of this equation? why not `this.readersCount * 0.019` instead?

**Review by williamchong — CHANGES_REQUESTED** [review]

**Review by williamchong — APPROVED** [review]

**Review by williamchong — APPROVED** [review]

**Comment by nwingt** [comment]
Merged into master

**Pull request closed without merging**